### PR TITLE
Make OpenStack database dump name more generic

### PR DESCRIPTION
**Pull request by rsalevsky**

### CODE REVIEW

File: chef/data_bags/crowbar/template-crowbar.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
       },
       "simple_proposal_ui": true,
       "upgrade": {
-        "db_dump_location" : "/var/lib/pgsql/db-before-cloud6.dump"
+        "db_dump_location" : "/var/lib/pgsql/upgrade-openstack-database.dump"
```
Comment (sayalilunkad) [inline chef/data_bags/crowbar/template-crowbar.json]: This name doesn't really make sense. Would be better to use something like openstack-db-pre-upgrade.dump

Comment (nicolasbock) [inline chef/data_bags/crowbar/template-crowbar.json]: Or `openstack-db-before-upgrade.dump`?

**Comment by sayalilunkad** [comment]
+1, thanks!

**Comment by jsuchome** [comment]
+1

**Comment by nicolasbock** [comment]
+1

**Comment by vuntz** [comment]
Don't we need a migration to have the new name used in existing proposals?

**Comment by nicolasbock** [comment]
Yes you are right, I missed that :(

**Pull request merged**